src/libbson/src/bson/bson-clock.c

```diff
@@ -16,10 +16,6 @@
 
 
 #ifdef __APPLE__
```

Comment:
This entire `#ifdef` can be removed if it no longer has a body.

Reply:
I thought I already did this and I have no idea why that commit wasn't showing before. I just made a new commit ("remove empty line") and now the "remove #ifdef" commit is showing :man-shrugging: